plugin/core/views.py

```diff
@@ -172,3 +174,22 @@ def text_document_range_formatting(view: sublime.View, region: sublime.Region) -
         "options": formatting_options(view.settings()),
         "range": region_to_range(view, region).to_lsp()
     })
+
+
+def minihtml(content: Union[str, dict], view: sublime.View) -> str:
+    """ Content can be a string or MarkupContent """
+    if isinstance(content, str):
+        return html.escape(content)
+
```

Comment:
I understand that some use this style of have an empty line in if blocks but I find it weird :)

Reply:
I agree. The liberal usage of empty lines is too much for me. The fact that PEP8 enforces empty lines between functions and classes provides enough visual separation.

If you believe your function is too hard to understand without empty lines, you should split up your function into smaller function. Enter PEP8 to enforce empty lines :)